cmd/odf/set/log_level.go

```diff
@@ -0,0 +1,50 @@
+package set
+
+import (
+	"fmt"
+	"regexp"
+
+	root "github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/red-hat-storage/odf-cli/pkg/rook/common"
+	"github.com/spf13/cobra"
+)
+
+var logLevelCmd = &cobra.Command{
+	Use:                "log-level",
+	Short:              "Set different log levels for ceph dameons like mon, osd and mds",
+	Example:            "odf set log-level osd 20",
```

Comment:
```suggestion
	Example:            "odf set log-level osd 20",
```

adding example looks nice